### ISSUE TITLE
How to obtain for each joint their global Orientation

### ISSUE THREAD

**Issue by benMOOVENCY** [mks0601/I2L-MeshNet_RELEASE#81]
Hello, 
I search to find for each joint their orientation in the word. 
If i understand well the  I obtain the relativ orientation for each joint  from this line in model.py with the pose_param.

# parameter regression
            pose_param, shape_param = self.param_regressor(joint_img_from_mesh.detach())

this orientation are relative to the parent joint and to the Initial T pose (template pose), have right ? 

How can i transform this data to the global orientation ?

**Comment by mks0601** [comment]
See my another codes: https://github.com/facebookresearch/DeepHandMesh/blob/e08dfc4cb69b2cb99f73fdefefff975d68a5bcc9/common/utils/transforms.py#L28

**Comment by benMOOVENCY** [comment]
Thank you for the answer, I probably didn't understand or see which part of the code allowed to calculate the global orientation of each joint, independently of the initial position?

**Comment by mks0601** [comment]
You can refer to forward_kinematics function. It recursively multiply 4x4 (rotation+translation) matrices.
